mdata/chunk/chunk.go

```diff
@@ -38,11 +38,15 @@ func (c *Chunk) Push(t uint32, v float64) error {
 	c.Series.Push(t, v)
 	c.NumPoints += 1
 	c.LastTs = t
-	c.LastWrite = uint32(time.Now().Unix())
 	totalPoints.Inc()
 	return nil
 }
 
 func (c *Chunk) Clear() {
 	totalPoints.DecUint64(uint64(c.NumPoints))
 }
+
+func (c *Chunk) Finish() {
+	c.Closed = true
```

Comment:
Is the reason why we don't need locks around these chunk properties because we relay on `AggMetric` to put locks around all operations on chunks?